lib/tasks/linksf.rake

```diff
@@ -42,10 +42,19 @@ namespace :linksf do
       category_name = record[:categories]
       category_name = 'shelter' if category_name == 'housing'
 
+      puts 'adding ' + resource.name
+
+      puts 'resource description is :' + resource.long_description
+
       record[:services].each do |service_json|
         category_name = service_json[:category]
         category_name = 'shelter' if category_name == 'housing'
         cat = Category.where('lower(name) = ?', category_name).first
+
+        if resource.long_description.blank? || resource.long_description.length < 15
+          puts 'replacing bad description' + resource.long_description + ' with ' + service_json[:description]
```

Comment:
NBD since this is just informational, but there's a missing space after "bad description".

Also, it looks like we're not doing any replacing here, just printing this out. Might be worth bringing up #106 as an implementation of #97 if the two approaches are of comparable implementation cost.